docs/novnc_server.1

```diff
@@ -0,0 +1,32 @@
+.TH novnc_server 1  "July 14, 2020" "version 1.2.0" "USER COMMANDS"
+
+.SH NAME
+novnc_server - noVNC proxy server
```

Comment:
I'd consider renaming it to novnc_proxy, otherwise, people might think this will launch a VNC server.

Reply:
Done.